docs/tutorials/intro_to_chemistry.ipynb

```diff
@@ -0,0 +1,310 @@
+        "* Note that elementary particles like electron carry an intrinsic form of angular momentum called Spin, which cannot be described by a function of positional variables $\\vec{r}$, and is somewhat analogous to earth rotating around its own axis.\n",
+        "* Since electrons (fermions) are indistinguishable, the wavefunction $\\Psi(\\vec{x_1}, \\vec{x_2}, ...)$ must be anti-symmetric to satisfy Pauli's Exclusion Principle.\n",
+        "\n",
+        "### Molecular Hamiltonian\n",
+        "$$\\hat{H} = \\underset{\\text{Nuclear K.E.}}{\\hat{T}_N(R)} + \\underset{\\text{Electronic K.E.}}{\\hat{T}_e(r)} + \\underset{\\text{Elec/Nuc attraction}}{\\hat{V}_{eN}(r, R)} + \\underset{\\text{Nuc/Nuc repulsion}}{\\hat{V}_{NN}(R)} + \\underset{\\text{Elec/Elec repulsion}}{\\hat{V}_{ee}(r)}$$\n",
+        "\n",
+        "### Born-Oppenheimer Approximation\n",
+        "* Since Nuclei are much larger than electrons, consider the nuclei to be fixed classical point charges. \n",
+        "* Therefore, Nuclear coordinates $R$ become fixed parameters; i.e. $\\hat{T}_N(R) \\approx 0$, $\\hat{V}_{NN}(R)$ is a constant and $\\hat{V}_{eN}(r; R)$ depends only parametrically on $R$.\n",
+        "* This results in Electronic Hamiltonian given as\n",
+        "\n",
+        "$$\\hat{H}_{el} = \\underset{\\text{One-Electron Operators}}{\\underset{\\text{Electronic K.E.}}{\\hat{T}_e(r)}  + \\underset{\\text{Elec/Nuc attraction}}{\\hat{V}_{eN}(r; R)}} + \\underset{\\text{Two-Electron Operator}}{\\underset{\\text{Elec/Elec repulsion}}{\\hat{V}_{ee}(r)}} = \\underset{\\text{One-Electron Operator}}{\\sum_{i}\\hat{h}(i)} + \\underset{\\text{Two-Electron Operator}}{\\sum_{i\\ne j}\\hat{v}(i, j)}$$\n",
+        "\n",
+        "### Potential Energy Surfaces\n",
+        "* For a given configuration of the Nuclei (fixed $R$), we aim to find the the energy eigenstates $|E_i\\rangle$ and the corresponding energy eigenvalues $E_i$ of the hamiltonian $H_e$.\n",
+        "* We can solve this equation for a range of nuclear configurations($R$) to map out the Potential Energy Surface of the molecule.\n",
+        "\n",
+        "### Orbitals and Chemical Basis Sets\n",
+        "$$\\underset{\\text{Spin Orbital}}{\\chi(\\vec{x})} = \\underset{\\text{Atomic Orbital}}{\\Phi(\\vec{r})} \\cdot \\underset{\\text{Spin Function}}{\\sigma(\\omega)} $$\n",
+        "* A Spin Orbital is a one-electron wavefunction which describes the position and spin of an electron in an atom. The spatial component of this one-electron function is called Atomic Orbital.\n",
+        "* A Basis Set is a set of functions, called basis functions, such that any elecronic wavefunction can be (approximately) represented as a linear combination of anti-symmetrized product of basis functions. \n",
+        "* Chemical basis sets are usually composed of spin orbital functions, examples of which include Gaussion-type Orbitals (GTO), Slater-type Orbitals (STO), Slater Type Orbital-n Gaussians (STO-nG) etc. \n",
```

Comment:
'Gaussian' (it's a proper name, so the capital is appropriate here).

Reply:
done